ui_framework/components/index.scss

```diff
@@ -94,6 +102,66 @@ $formTransitionTiming: 0.1s linear;
   }
 }
 
+@mixin bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+/**
+ * 1. Put 10px of space between each child.
+ * 2. If there is only one child, align it to the right. If you wanted it aligned right, you
+ *    wouldn't use the Bar in the first place.
+ */
+@mixin barSection {
+  display: flex;
+  align-items: center;
+  margin-left: $toolBarSectionSpacing * 0.5;
+  margin-right: $toolBarSectionSpacing * 0.5;
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &:only-child {
+    margin-left: auto; /* 1 */
+  }
+
+  & > * {
+    flex: 1 0 auto;
+  }
+
+  & > * + * {
+    margin-left: $toolBarItsemSpacing; /* 1 */
+  }
+}
+
+@mixin buttonOnReverseBackground {
+  .kuiButton--basic {
+    color: #5a5a5a;
+    background-color: #FFFFFF;
+
+    &:disabled {
+      color: #a7a7a7;
+      background-color: #F3F3F3;
+    }
+  }
+}
+
+@mixin selectOnReverseBackground {
```

Comment:
I see these `*OnReverseBackground` styles but in the UI Framework web site section for this component I see "This component is missing Dark Theme variations." Maybe that's still accurate but just wanted to bring it up in case it wasn't and needed updating.